trestle/core/commands/common/return_codes.py

```diff
@@ -41,3 +42,4 @@ class CmdReturnCodes(enum.Enum):
     IO_ERROR = 6
     AUTH_ERROR = 7
     UNKNOWN_ERROR = 8
+    INVALID_MODEL = 9
```

Comment:
Please use `OSCAL_VALIDATION_ERROR ` as it was created for this purpose

Reply:
done